GEM-data/gem-collections.ipynb

```diff
@@ -51,6 +51,36 @@
     "bbox = BBox(bbox=[-18, 1, 27.5, 28], crs=CRS.WGS84)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "e390008c-a274-46da-8521-b8f1e8c9e78a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def templat_evalscript(bands, sampleType):\n",
+    "    _input = str([band for band in bands]).replace(\"'\", '\"')\n",
```

Comment:
use `input_bands` or similar, the `_` prefix has a strong connotation in python, i think it's better if the names don't match 100% than to use it

Reply:
done